R/getNeatOrder.R

```diff
@@ -17,38 +17,64 @@
 #' allows for a more faithful representation of the data's intrinsic structure as captured 
 #' by the ordination process.
 #' 
-#' @param x A matrix containing the ordinated data to be sorted. Columns should represent the principal components (PCs) and rows should represent the entities being analyzed (e.g., features or samples).
-#' @param subset A vector specifying a subset of rows to be used and retained. If NULL, all rows are used.
-#' @param dimensions A vector of length 2 specifying the columns of the matrix to use for the X and Y coordinates.
-#' @param centering_method A character string specifying the method to center the data. Options are "mean", "median", or "none" if your data is already centred.
-#' @param decreasing A boolean that when true sorts the rows in a descending order by radial theta angle. Default is False.
+#' @param x A matrix containing the ordinated data to be sorted. Columns should represent the principal components (PCs) and rows should represent the entities being analyzed (e.g. features or samples).
+#' @param dimensions A \code{character} or \code{integer} vector of length 2 specifying the columns of the matrix to use for the X and Y coordinates.
```

Comment:
do we need this argument?

Reply:
I can remove decreasing if needed yeah, it would mean the sorting would only occur in one order but I can definitely do that

Reply:
@antagomir - just to confirm - you want me to remove decreasing, and should I remove the dimensions argument too?

Reply:
cause if I was to call like this - sorted_order <- getNeatOrder(reducedDim(tse, "PCA")[, c(1,2)], method = "mean") - that would mean removing the dimensions argument. Although I quite like it as an argument cause you can make it default (1,2) - but it's very easy to also just do it when you call the method

Reply:
The question is more like, this argument is not needed as 1) it is faster to write x[, c(1,2)] in input than "x, dimensions=c(1,2)" in the function call and 2) this will also simplify function and reduce maintenance burden (many small things like this will create a big task stream in the end)